app/routes/wow.item-data.$itemId.tsx

```
@@ -99,172 +168,191 @@
+              )}
+              {listing.avgQuantity !== -1 && (
+                <Differences
+                  diffTitle="Average Quantity"
+                  diffAmount={listing.avgQuantity.toLocaleString()}
+                  className="bg-blue-100 text-blue-900 font-semibold dark:bg-blue-600 dark:text-gray-100"
+                />
+              )}
+              {listing.currentVsAvgQuantityPercent !== 100 && (
+                <Differences
+                  diffTitle="Avg v Current Quantity"
+                  diffAmount={listing.currentVsAvgQuantityPercent.toLocaleString() + '%'}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)